docs/releases/3.2.txt

```diff
@@ -569,3 +569,10 @@ Miscellaneous
 * The ``default_app_config`` application configuration variable is deprecated,
   due to the now automatic ``AppConfig`` discovery. See :ref:`whats-new-3.2`
   for more details.
+
+* ``normalize_email()`` is now implemented in
+  :class:`~django.contrib.auth.models.AbstractUser` instead of
+  :class:`~django.contrib.auth.models.BaseUserManager`. Customized email
+  normalization should override
+  :meth:`~django.contrib.auth.models.AbstractUser.normalize_email`
+  before backwards compatibility is removed in Django 4.1.
```

Comment:
We don't mention the future versions things will be removed here because it's implied by the deprecation policy, noted in the warning message, and we might change our minds about the removed version.